### ISSUE TITLE
removal on npm2

### ISSUE THREAD

**Issue by psypersky** [developit/modify-babel-preset#13]
Looks like you already now about this, the test fails, could you point me in the right direction on why this happens and why does it only happens on npm 2? thanks

**Comment by developit** [comment]
I wish I knew more, basically the name lookup fails because it can't figure out what preset module to look under.

**Comment by developit** [comment]
For what it's worth, Babel is basically dropping support for npm 2. That that it's a big deal, just i know personally I was a holdout but have finally switched to 3.

**Comment by psypersky** [comment]
Ohh ok, thanks @developit ill update to npm 3 i hope i don't break other stuff doing that haha

**Comment by developit** [comment]
It's so much faster, you won't regret it!

**Comment by psypersky** [comment]
Yep, that fixed the problem, and indeed i can't go and watch a netflix episode anymore when doing "npm i" thanks man i have been trying to solve this for a while, maybe its worth to mention it on the readme so other do not fall in this problem

**Comment by developit** [comment]
For sure!
